hack/test/spider/main.go

```diff
@@ -0,0 +1,172 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"sort"
+	"strings"
+)
+
+var sha1, sha2, commitRange, filterDir string
```

Comment:
```suggestion
var shaA, shaB, commitRange, filterDir string
```
suggest using A and B to avoid confusion with SHA-1 and SHA-2 algorithms.